toxcore/tox.api.h

```diff
@@ -286,7 +286,10 @@ const FILE_ID_LENGTH              = 32;
  */
 const MAX_FILENAME_LENGTH         = 255;
 
-
+/**
+ * Max legnth of names queried by toxcore
+ */
+const MAX_QUERY_NAME_LENGTH       = 255;
```

Comment:
It's already limited to 63 chars in worst-case scenario. For 64 you would want 256.